interface/patient_file/download_template.php

```diff
@@ -357,44 +357,35 @@ function doSubs($s)
 $fname = tempnam($GLOBALS['temporary_files_dir'], 'OED');
 
 // Get mime type in a way that works with old and new PHP releases.
-$mimetype = 'application/octet-stream';
+$default_mimetype = 'application/octet-stream';
 $ext = strtolower(array_pop(explode('.', (string) $form_filename)));
-if ('dotx' == $ext) {
-    // PHP does not seem to recognize this type.
-    $mimetype = 'application/msword';
```

Comment:
i think this is getting missed below:
$mimetype = 'application/msword'; for 'dotx'

Reply:
disregard, i see it now :)